arm-notificationhubs/2014-09-01/swagger/notificationhubs.json

```diff
@@ -1660,33 +1654,33 @@
       "properties": {
         "id": {
           "type": "string",
-          "description": "Gets or sets the id of the created NotificatioHub."
+          "description": "Gets or sets the id of the created NotificationHub."
```

Comment:
You should just say "The id of the created NotificationHub ." -- individual language generators in AutorRest will add the Gets or Sets if it's appropriate (some languages don't support properties)